binding/Binding/SKTypeface.cs

```diff
@@ -16,6 +17,7 @@ public class SKTypeface : SKObject
 		internal SKTypeface (IntPtr handle)
 			: base (handle)
 		{
+			if (handle == IntPtr.Zero) throw new Exception("Invalid typeface");
```

Comment:
Hi @mattleibow 
That's ok, I have removed the check from the SKTypeface constructor taking an IntPtr handle.